src/elements/content-uploader/UploadState.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import { HatWand } from '@box/blueprint-web-assets/illustrations/Medium';
+
+import UploadEmptyState from '../../icons/states/UploadEmptyState';
+import UploadSuccessState from '../../icons/states/UploadSuccessState';
+import UploadStateContent from './UploadStateContent';
```

Comment:
created ticket to replace UploadSuccessState and UploadStateContent icons in later state since there is currently no matching illustrations in blueprint.
https://jira.inside-box.net/browse/DEVECO-10393